--- binutils-2.20/gas/config/tc-arm.c	2009-08-30 00:10:59.000000000 +0200
+++ binutils-2.20_ok/gas/config/tc-arm.c	2009-10-20 11:08:05.000000000 +0200
@@ -2486,7 +2486,9 @@ make_mapping_symbol (enum mstate state, 
       frag->tc_frag_data.first_map = symbolP;
     }
   if (frag->tc_frag_data.last_map != NULL)
+  {
     know (S_GET_VALUE (frag->tc_frag_data.last_map) < S_GET_VALUE (symbolP));
+  }
   frag->tc_frag_data.last_map = symbolP;
 }
 
